tests/tools/create_dummy_box.sh

```diff
@@ -8,15 +8,15 @@ if [ $# -ne 1 ]; then
 	exit 1
 fi
 
-if [ -f "$DIR/dummy.box" ]; then
+if [ -f "${DIR}/dummy.box" ]; then
```

Comment:
Not related to this PR but looks like I produced a broken check. Should have been dummy-$provider.box not dummy.box.

Reply:
Updated.